pyproject.toml

```diff
@@ -106,8 +106,13 @@ lint.isort.required-imports = ["from __future__ import annotations"]
 "__main__.py" = ["T20"]
 
 [tool.pytest.ini_options]
-addopts = "-v -ra"
+addopts = ["-v", "-ra", "--strict-config", "--strict-markers"]
 norecursedirs = ["src"]
+minversion = "9"
```

Comment:
If we require 9, we could go ahead and use the new nicer configuration style (`[tool.pytest]`, `strict=true`, etc).